deployment/gentoo-linux/kleiner-brauhelfer-2.6.1.ebuild

```diff
@@ -0,0 +1,44 @@
+EAPI=7
+inherit desktop qmake-utils
+DESCRIPTION="Der kleine Brauhelfer ist ein Hilfsprogramm für Hobbybrauer zum Erstellen und Verwalten von Biersuden"
+HOMEPAGE="https://github.com/kleiner-brauhelfer/kleiner-brauhelfer-2"
```

Comment:
Alternative, you may also linkt to http://kleiner-brauhelfer.github.io

Reply:
Done